Library/Formula/x264.rb

```diff
@@ -43,7 +43,7 @@ def install
     elsif Formula["gpac"].installed?
       args << "--disable-lsmash"
     end
-    args << "--bit-depth=10" if build.include? "10-bit"
+    args << "--bit-depth=10" if build.include? "with-10-bit"
```

Comment:
Quite right. I was just aiming to keep the structure the same for those who may be keeping that file updated regularly.